crates/qt-build-utils/src/lib.rs

```diff
@@ -245,11 +246,37 @@ impl QtBuild {
         };
 
         for qt_module in &self.qt_modules {
-            println!("cargo:rustc-link-lib=Qt{}{}", self.version.major, qt_module);
-            let prl_path = format!(
-                "{}/{}Qt{}{}.prl",
-                lib_path, prefix, self.version.major, qt_module
-            );
+            let framework = match &target {
+                Ok(target) => {
+                    if target.contains("apple") {
+                        Path::new(&format!("{}/Qt{}.framework", lib_path, qt_module)).exists()
+                    } else {
+                        false
+                    }
+                }
+                Err(_) => false,
+            };
+
+            let (link_lib, prl_path) = if framework {
+                (
+                    format!("framework=Qt{}", qt_module),
+                    format!(
+                        "{}/Qt{}.framework/Resources/Qt{}.prl",
```

Comment:
so in the Qt code it searches the Resources folder first then searches the framework folder for backwards compatibility. This would be for < 5.14 support.

As we only support 5.15+ we can ignore this I guess but something to note and could be fixed separately in qt-build-utils.